Tests/govuk_ios/govuk_ios_unit_tests/Mocks/AppConfig/MockAppConfigProvider.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+class MockAppConfigProvider: AppConfigProviderInterface {
+
+    var _receivedAppConfigCompletion:
+    ((Result<AppConfig, AppConfigServiceError>) -> Void)?
+
+    func fetchAppConfig(filename: String ,completionHandler: @escaping (Result<AppConfig, AppConfigServiceError>) -> Void) {
```

Comment:
`func fetchAppConfig(filename: String , ` > ` func fetchAppConfig(filename: String, ` + maybe knock the completionHandler arg to a new line